lib/python/treadmill_aws/cli/__init__.py

```diff
@@ -84,6 +84,21 @@ def sanitize_user_name(ctx, param, value):
     return value.lower()
 
 
+def sanitize_partition_name(ctx, param, value):
```

Comment:
this should be reverse.

if value == _default:
   return None

Reply:
If the user does not pass a partition name (value==None) the regex on L96 will error with a TypeError.
PR updated; is this an acceptable compromise?